specification/cognitiveservices/data-plane/Personalizer/preview/v1.1-preview.2/Personalizer.json

```diff
@@ -0,0 +1,1895 @@
+        "description": "Report that the specified event was actually used (e.g. by being displayed to the user) and a reward should be expected for it.",
+        "operationId": "Events_Activate",
+        "consumes": [],
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "name": "eventId",
+            "in": "path",
+            "description": "The event ID to be activated.",
+            "required": true,
+            "type": "string",
+            "maxLength": 256
+          }
+        ],
+        "responses": {
+          "204": {
+            "description": "Success"
+          },
+          "default": {
+            "description": "Invalid activate event request.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Successful Events_Activate request": {
+            "$ref": "./examples/Events_Activate.json"
+          }
+        }
+      }
+    },
+    "/configurations/applyFromEvaluation": {
+      "post": {
+        "tags": [
+          "ConfigurationsV1Dot1Preview1"
+        ],
+        "summary": "Apply Learning Settings and model from a pre-existing Offline Evaluation, making them the current online Learning Settings and model and replacing the previous ones.",
+        "operationId": "ServiceConfiguration_ApplyFromEvaluation",
+        "consumes": [
+          "application/json"
+        ],
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "in": "body",
+            "name": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/PolicyReferenceContract"
+            }
+          }
+        ],
+        "responses": {
+          "204": {
+            "description": "Success"
+          },
+          "default": {
+            "description": "Learning Settings not found in evaluation.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Successful ServiceConfiguration_ApplyFromEvaluation request": {
+            "$ref": "./examples/ServiceConfiguration_ApplyFromEvaluation.json"
+          }
+        }
+      }
+    },
+    "/logs/interactions": {
+      "post": {
+        "tags": [
+          "LogsV1Dot1Preview2"
+        ],
+        "summary": "Post interactions.",
+        "description": "The endpoint is intended to be used from within a SDK for logging interactions and accepts specific format defined in https://github.com/VowpalWabbit/reinforcement_learning. This endpoint should not be used by the customer.",
+        "operationId": "Log_Interactions",
+        "consumes": [
+          "application/octet-stream"
+        ],
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "in": "body",
```

Comment:
@slahabar -- Please check that your parameters have descriptions as well.

Reply:
Added description for all parameters.